src/OpenDiffix.Core/Expression.fs

```diff
@@ -204,6 +204,10 @@ let rec evaluateScalarFunction fn args =
   | Cast, [ Real r; String "text" ] -> r.ToString(doubleStyle) |> String
   | Cast, [ Boolean b; String "text" ] -> b.ToString().ToLower() |> String
 
+  | Cast, [ Integer i; String "integer" ] -> Integer i
```

Comment:
I would add this to line 199 alongside numeric casts.

Reply:
I'm not sure what pattern/ordering of clauses you are after? I don't recognize any pattern in the preexisting clauses, so the new clauses were added at the end and clustered together (as they all fall into the "noop cast" category).

Reply:
Ok with the above pattern it looks good. I thought they were grouped by source type.